lib/modules/onc_program/bulk_data_export_sequence.rb

```diff
@@ -54,7 +54,7 @@ def check_capability_statement
           break if operation.present?
         end
 
-        assert operation.present?, 'Server CapabilityStatement did not declare export operation in Group resource.'
+        assert operation.present?, 'Server CapabilityStatement did not declare support of export operation in Group resource.'
```

Comment:
I think these should all be `support for`.

Reply:
OK. "support for"